guides/common/modules/proc_managing-ostree-content-with-content-views.adoc

```diff
@@ -0,0 +1,55 @@
+[id="managing-ostree-content-with-content-views_{context}"]
+= Managing OSTree Content with Content Views
+
+Use Content Views to manage OSTree branches across the application life cycle.
+This process uses the same publication and promotion method as RPMs or Puppet modules.
+
+.Prerequisites
+
+* ??? _Are there any?_
```

Comment:
I suppose you would need available repositories of synced OSTree content? 
For more info see Importing OSTree content? 
Something like that?